es6-destructuring/index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+<script>
+  const book1 = {
+    title: 'Goodnight Punpun',
+    author: 'Inio Asano',
+    libraryID: 3353
+  };
+
+  const {
+    title,
+    author,
+    libraryID
+  } = book1;
+  console.log(`The title of the book is ${title}, the author is ${author}, and the libary id is ${libraryID}`);
+
+  const book2 = {
+    titleTwo: 'Les Fleurs du mal',
+    authorTwo: 'Charles Baudelaire',
+    libraryIDTwo: 2345
+  };
+
+  const {
+    titleTwo,
+    authorTwo,
+    libraryIDTwo
+  } = book2;
```

Comment:
For this part we want to use destructured aliasing. The properties should be the same as the first book, but the destructured assignment should use alternate names.

```js
    const { title: title2, author: author2, libraryID: libID2 } = book2;
```